lib/checks/landmarks/landmark-is-unique-after.js

```diff
@@ -0,0 +1,19 @@
+const uniqueLandmarks = [];
+results.forEach(currentResult => {
+	const findMatch = someResult => {
+		return (
+			currentResult.data.role === someResult.data.role &&
+			currentResult.data.label === someResult.data.label
+		);
+	};
+
+	const foundResultIndex = uniqueLandmarks.findIndex(findMatch);
+	if (foundResultIndex === -1) {
+		uniqueLandmarks.push(currentResult);
+	} else {
+		currentResult.result = false;
```

Comment:
We should be adding the information about the currentResult to the matching item as a `relatedNode` that way we only return one fail for each duplicated role and name and we can see the others directly in the result.

Reply:
@WilcoFiers I believe that we can build a list of related nodes for each element and the call the result object's `relatedNodes` helper function to turn it into the correct type of array - can you confirm?

Reply:
I followed the pattern set by access-keys; let me know what you think 😄